.github/workflows/main.yml

```diff
@@ -69,4 +77,17 @@ jobs:
     - name: Publish results to coveralls
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        COVERALLS_PARALLEL: true
       run: coveralls --rcfile=pyproject.toml --service=github
```

Comment:
Probably need to update this to use Coveralls Action.

Reply:
Can I have thoughts later?  Revert to whatever works for now.  I'll make an issue.